configure.ac

```diff
@@ -200,16 +200,13 @@ AM_CONDITIONAL([BUILD_ALL], [test "x$ENABLED_ALL" = "xyes"])
 
 # Support for forcing 32-bit mode
 AC_ARG_ENABLE([32bit],
-    [AS_HELP_STRING([--enable-32bit],[Enables 32-bit support (default: disabled)])],
+    [AS_HELP_STRING([--enable-32bit],[Enables forced 32-bit instruction (default: disabled)])],
```

Comment:
Should we remove the 32bit option altogether?

Reply:
I did some further cleanup here. Decided to keep the option and use it to disable algos that require 64-bit types such as SHA512/384, SHA-3 and Ed/Curve25519 (this was existing behavior). The PR just removes the `-m32` and `NO_64BIT`. Updated PR description and commit to reflect this.